Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -479,41 +344,21 @@ def _on_selectionChanged(self):
             return
 
         areas = self.vennwidget.vennareas()
-        indices = [i for i, area in enumerate(areas)
-                   if area.isSelected()]
-
-        self.selection = indices
-
+        self.selection = [i for i, area in enumerate(areas) if area.isSelected()]
         self.invalidateOutput()
 
-    def _on_useidentifiersChanged(self):
-        self.inputsBox.setEnabled(self.useidentifiers == 1)
-        # Invalidate all itemsets
-        self._invalidate()
-        self._updateItemsets()
+    def _on_matching_changed(self):
+        self.output_duplicates_box.setEnabled(bool(self.rowwise))
```

Comment:
Errrr, I don't get why the whole Output box would be disabled for Columns - including autocommit!
Probably only the `Output duplicates` option should be?

Edit: maybe this bug also hints at non-optimal naming - `output_duplicates_box` could be just `output_box` (duplicates setting is only one part of it).